src/main/java/org/elasticsearch/index/mapper/internal/VersionFieldMapper.java

```diff
@@ -96,7 +96,7 @@ protected Field initialValue() {
     };
 
     public VersionFieldMapper(Settings indexSettings) {
-        super(new Names(NAME, NAME, NAME, NAME), Defaults.BOOST, Defaults.FIELD_TYPE, null, null, null, null, null, null, indexSettings);
+        super(new Names(NAME, NAME, NAME, NAME), Defaults.BOOST, Defaults.FIELD_TYPE, false, null, null, null, null, null, indexSettings);
```

Comment:
Which makes me wonder that we should probably a test that the default mapping serializes to an empty json to make sure that this default logic works correctly?